### DISCUSSION TITLE
projection data file

### DISCUSSION THREAD

**Discussion by gnomus-maximus** [neutronimaging/imagingsuite#572] [Q&A]
I#m trying to enter my projection data with a .txt file. Wenn i do so i get the error messsage: _GetTIFFDims: Could not open_. Is this a knowen issue. What have the projection data file look like an how do i enter it proborly? Is there a exampel of how to use a file to enter data. 

thank you for any help, best reguard, Clemens

**Comment by anderskaestner** [comment]
This is a long time ago... did you find a solution?
